teams/Dependancies.toml

```diff
@@ -0,0 +1,49 @@
+[[dependency]]
```

Comment:
Check the name of the file. There is a typo
It should be Dependencies.toml

Reply:
Fixed this thanks